Scripts/Python/ki/xKIChat.py

```diff
@@ -1435,3 +1436,42 @@ def MarkerGameReward(self, reward):
             self.chatMgr.AddChatLine(None, "Current Reward: '{}'".format(markerMgr.reward), 0)
         else:
             self.chatMgr.AddChatLine(None, "This game has no associated reward.", 0)
+
+    ## Rolls the specified dice and displays the results
+    def RollDice(self, dice_str):
+        if not dice_str:
+            self.chatMgr.AddChatLine(None, "Please provide a number of dice and number of faces in common dice notation (e.g. 2d6), or 'fate' to roll a set of fate dice.", kChat.SystemMessage)
```

Comment:
Why not simply roll one cuboid die by default?

Reply:
I've been debating that.  I weighed it against the utility of a self-documenting help response.  Not sure which is best here.  Error message or automatic 1d6.

Reply:
From the perspective of someone who does not play tabletop games, I would have had no idea how to "provide a number of dice and number of faces" (until yesterday's research) and would expect rolling a die to yield a single number 1-6. :wink: